src/components/ScrollWrapper/hooks/useScrollWrapper.ts

```diff
@@ -0,0 +1,37 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { RefObject, useState, UIEvent } from "react";
+
+export const useScrollWrapper = (scrollingContainer: RefObject<HTMLElement>) => {
+    const [scrollDimensions, setScrollDimensions] = useState({ top: 0, height: Infinity, width: Infinity, left: 0 });
+
+    const onScroll = (event: UIEvent) => {
+        const target = event.target as HTMLElement;
+        setScrollDimensions({
+            top: target.scrollTop,
+            height: target.scrollHeight,
+            left: target.scrollLeft,
+            width: target.scrollWidth,
+        });
+    };
+
```

Comment:
add useEffect to set the scroll wrapper values once the element has loaded, incase area is too small to scroll and having the height set to infinity will cause the bottom shadow to permanently appear